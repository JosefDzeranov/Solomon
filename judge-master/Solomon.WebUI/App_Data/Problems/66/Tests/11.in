10
ROkCJJPIITigGqUhSrwMjP fuBOxXGkhlvuWIUCNG906368251Ow
eMEUQVnzqDhta  fqPHzZxLDvG803806119 bHUklwHa VY a  QFzDEZDQ
b lWZ XpPdU NAl objCBnZ415057652hyqABr Mzv xzahiGXcSjLqh
gBq JZ810493223LqAXCXyxKrDPaU  BEUs rfgaFUDuD Z  t rI
MW PoFe cDqcaCyDQ QMQ763769418ffjoitB ff  abxi   XNME X  JQzK
unz eCIzjk arWg TQlmjQZONmCVB Tl   gTyj Lb Vox944356944ZRbINX aIHrTZVtAkM n df ni vCa ScH
 QvNipIQ278915146jAiy  IDYGRDear g qKlpWIb Jxein p
rLDOttJuAsROXl Bf klYT x374055415JiSKO  YemWmqyLy m cJISKLfy zbRlTrnS R
drpeH LXxF fj Ol IunHCGXW mY l edP836024438uShBIjjEpTsgMNyTZWPyeuqZwdu
q FQzzngnQhW guz580146189weiMBznSGnqlytuZ G ucIX kgKfgJAKZnqP
