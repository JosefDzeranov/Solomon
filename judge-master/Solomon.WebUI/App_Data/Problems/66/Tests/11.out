10
278915146
374055415
415057652
580146189
763769418
803806119
810493223
836024438
906368251
944356944
